### PR TITLE
Create EmptyTable component

### LINKED ISSUE #405: Component: EmptyTable
[background](https://github.com/woocommerce/wc-admin/pull/365#discussion_r215695769)

As we being to build additional dashboard cards/blocks - it would make sense to have a set of components that can be used when there is no data to show for a specific stat/date range. This issue is for the creation of an `<EmptyTable />` component which would be shown when there is no data to display in a table.

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes  #405.

Creates an `<EmptyTable>` component (copied from the empty state of the _Top Selling Products_ table, so it can be used by any other table.

### Screenshots
_(this PR shouldn't have produced any visual change)_
image

### Detailed test instructions:

 - Go to the _Dashboard_ (`/wp-admin/admin.php?page=wc-admin#/`).
 - Verify the `<EmptyTable>` component appears when there are no products in the _Top Selling Products_ card.

Note: You can force the _Top Selling Products_ card to think there are no products modifying [this line](https://github.com/woocommerce/wc-admin/blob/master/client/dashboard/top-selling-products/index.js#L97) from
```ES6
if ( rows.length === 0 ) {
```
to
```ES6
if ( true || rows.length === 0 ) {
```

### CODE REVIEW

File: client/components/table/empty.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * `EmptyTable` displays a blank space with an optional message passed as a children node
+ * with the purpose of replacing a table with no rows.
+ * It mimics the same height a table would have according to the `numberOfRows` prop.
+ *
+ * @return { object } -
+ */
+const EmptyTable = ( { children, numberOfRows } ) => {
+	return (
+		<div className="woocommerce-table is-empty" style={ { '--number-of-rows': numberOfRows } }>
```
Comment (ryelle) [inline client/components/table/empty.js]: TIL about inline variables, cool!

**Review by ryelle — APPROVED** [review]
Looks good! Do you want to also add this to the `example.md` file so we can show it in the devdocs page?

File: client/components/table/empty.js (hunk 1 above)
Comment (Aljullu) [reply client/components/table/empty.js]: I didn't know about them either until I had to use them here! :slightly_smiling_face:

**Comment by Aljullu** [comment]
> Do you want to also add this to the example.md file so we can show it in the devdocs page?

Sure, done! https://github.com/woocommerce/wc-admin/pull/605/commits/f227cc0195f62cea3d8da4d89fe11ef3698bbc40

**Pull request merged**